### ISSUE TITLE
is there a way to run a command automatically when piping a message?

### ISSUE THREAD

**Issue by kurokirasama** [ThomasHabets/cmdg#101]
I need to run a command before I type my own when piping to a message.

More precisely, I'm using nushell as my shell, and I would like to load some custom command first so they are available.

**Comment by ThomasHabets** [comment]
No. If it's things that can't be loaded always (some rc file), can you just pipe it to a wrapper script that loads the stuff?

**Comment by kurokirasama** [comment]
I'll try, thanks
